src/systems/brush.js

```diff
@@ -229,6 +245,19 @@ AFRAME.registerSystem('brush', {
 
     return stroke;
   },
+  createStrokeId: function () {
+    return Math.random().toString(36).substring(2, 9);
+  },
+  addPointToStroke: function (strokeId, data) {
+    var stroke;
+    for (var i = 0; i < this.strokes.length; i++) {
```

Comment:
What about having a map from strokeId and the strokes to avoid looking for the stroke on every tick when painting?